src/libs/Localize/index.ts

```diff
@@ -46,7 +46,13 @@ function init() {
     }, {});
 }
 
-type PhraseParameters<T> = T extends (...args: infer A) => string ? A : never[];
+type PluralFormPhrase = Record<string, string>;
+type PhraseParameters<T> = T extends (args: number | Record<string, string | number | undefined>) => string
+    ? [number | Record<string, string | number | undefined>]
+    : T extends (...args: infer A) => unknown
+    ? [...A]
+    : [];
```

Comment:
Actually, wouldn't the old version of this type work?
```suggestion
type PhraseParameters<T> = T extends (...args: infer A) => string ? A : never[];
```

Reply:
But we are inferring from `PhraseParameters` type argument, which is always `Localize.Phrase<TKey>`, so it's not like people can pass whatever they want here 🤔 

I also doesn't understand why `T extends (args: number | Record<string, string | number | undefined>) => string
    ? [number | Record<string, string | number | undefined>]` is followed with `T extends (...args: infer A) => unknown`. This makes the type a bit unreadable – it's either overcomplicated or I just don't understand it 😅

Reply:
I'm OOO, will get back to this on Monday. Sorry!